requirements.txt

```diff
@@ -1,9 +1,9 @@
 appdirs~=1.4
 cached-property~=1.4.3
 cli-helpers~=1.0.2
-git+https://gitlab.com/coala/coala-utils#egg=coala-utils-0.7.0
+coala-utils==0.7.0
```

Comment:
No, we will have to use git source for this.

Reply:
pytest-reqs fails to recognise it as an requirement.